egs/dihard_2018/v2/run.sh

```diff
@@ -0,0 +1,323 @@
+      | transform-vec $nnet_dir/xvectors_dihard_2018_dev/transform.mat ark:- ark:- \
+      | ivector-normalize-length ark:- ark:- |" \
+    $nnet_dir/xvectors_dihard_2018_dev/plda || exit 1;
+fi
+
+# Perform PLDA scoring
+if [ $stage -le 11 ]; then
+  # Perform PLDA scoring on all pairs of segments for each recording.
+  diarization/nnet3/xvector/score_plda.sh --cmd "$train_cmd --mem 4G" \
+    --nj 20 $nnet_dir/xvectors_dihard_2018_dev $nnet_dir/xvectors_dihard_2018_dev \
+    $nnet_dir/xvectors_dihard_2018_dev/plda_scores
+
+  diarization/nnet3/xvector/score_plda.sh --cmd "$train_cmd --mem 4G" \
+    --nj 20 $nnet_dir/xvectors_dihard_2018_dev $nnet_dir/xvectors_dihard_2018_eval \
+    $nnet_dir/xvectors_dihard_2018_eval/plda_scores
+fi
+
+# Cluster the PLDA scores using a stopping threshold.
+if [ $stage -le 12 ]; then
+  # First, we find the threshold that minimizes the DER on dihard_2018_dev.
+  mkdir -p $nnet_dir/tuning
+  echo "Tuning clustering threshold for dihard_2018_dev"
+  best_der=100
+  best_threshold=0
+
+  # The threshold is in terms of the log likelihood ratio provided by the
+  # PLDA scores.  In a perfectly calibrated system, the threshold is 0.
+  # In the following loop, we evaluate DER performance on dihard_2018_dev 
+  # using some reasonable thresholds for a well-calibrated system.
+  for threshold in -0.5 -0.4 -0.3 -0.2 -0.1 -0.05 0 0.05 0.1 0.2 0.3 0.4 0.5; do
+    diarization/cluster.sh --cmd "$train_cmd --mem 4G" --nj 20 \
+      --threshold $threshold $nnet_dir/xvectors_dihard_2018_dev/plda_scores \
+      $nnet_dir/xvectors_dihard_2018_dev/plda_scores_t$threshold
+
+    md-eval.pl -1 -c 0.25 -r data/dihard_2018_dev/rttm \
+     -s $nnet_dir/xvectors_dihard_2018_dev/plda_scores_t$threshold/rttm \
+     2> $nnet_dir/tuning/dihard_2018_dev_t${threshold}.log \
+     > $nnet_dir/tuning/dihard_2018_dev_t${threshold}
+
+    der=$(grep -oP 'DIARIZATION\ ERROR\ =\ \K[0-9]+([.][0-9]+)?' \
+      $nnet_dir/tuning/dihard_2018_dev_t${threshold})
+    if [ $(echo $der'<'$best_der | bc -l) -eq 1 ]; then
+      best_der=$der
+      best_threshold=$threshold
+    fi
+  done
+  echo "$best_threshold" > $nnet_dir/tuning/dihard_2018_dev_best
+
+  diarization/cluster.sh --cmd "$train_cmd --mem 4G" --nj 20 \
+    --threshold $(cat $nnet_dir/tuning/dihard_2018_dev_best) \
+    $nnet_dir/xvectors_dihard_2018_dev/plda_scores $nnet_dir/xvectors_dihard_2018_dev/plda_scores
+
+  # Cluster dihard_2018_eval using the best threshold found for xvectors_dihard_2018_dev. 
+  # xvectors_dihard_2018_dev is used as the dataset to tune the parameters. 
+  diarization/cluster.sh --cmd "$train_cmd --mem 4G" --nj 20 \
+    --threshold $(cat $nnet_dir/tuning/dihard_2018_dev_best) \
+    $nnet_dir/xvectors_dihard_2018_eval/plda_scores $nnet_dir/xvectors_dihard_2018_eval/plda_scores
+
+  mkdir -p $nnet_dir/results
+  # Compute the DER on the dihard_2018_eval. 250ms of a speaker transition is tolerated
+  # and overlapping segments are ignored. 
```

Comment:
If you want to include these results, can you clarify in the comment that this is different from how the DIHARD challenge was originally evaluated?